lib/active_model/serializer.rb

```diff
@@ -476,9 +479,7 @@ def scope
 
     def _serializable_hash
       return nil if @object.nil?
-      @node = attributes
```

Comment:
@bf4 the specs seem to fail randomly trying to call  `read_attribute_for_serialization` - the failures are not tied to versions of rails or ruby, and can change per run. The issue appears to be in the generated faker data, but I'm not exactly sure what's going on there.

Anyway, we've used this in production without issue since I opened this PR, so it's safe to merge.